app/models/mhv_account.rb

```diff
@@ -56,8 +67,11 @@ class MhvAccount < ActiveRecord::Base
   end
 
   def create_and_upgrade!
-    create_mhv_account! unless preexisting_account?
-    upgrade_mhv_account!
+    # Accept T&C if existing?
+    unless existing?
+      create_mhv_account! unless previously_registered?
```

Comment:
existing means the account has an mhv_correlation_id and previously_registered? means that vets.gov registered it? I would like for these method names to be a little more clear to that point maybe. It's a little confusing to think about it, but I think I get the idea.